pkg/commands/analyse_grafana.go

```diff
@@ -35,15 +41,20 @@ func (cmd *GrafanaAnalyseCommand) run(k *kingpin.ParseContext) error {
 		return err
 	}
 
-	boardLinks, err := c.SearchDashboards(ctx, "", false)
+	boardLinks, err = c.SearchDashboards(ctx, "", false)
 	if err != nil {
 		return err
 	}
 
 	for _, link := range boardLinks {
-		board, _, err := c.GetDashboardByUID(ctx, link.UID)
+		rawBoard, _, err = c.GetRawDashboardBySlug(ctx, link.URI)
 		if err != nil {
-			fmt.Fprintf(os.Stderr, "%s for %s %s\n", err, link.UID, link.Title)
+			fmt.Fprintf(os.Stderr, "%s for %s\n", err, link.URI)
+			continue
+		}
+
+		if err = json.Unmarshal(rawBoard, &board); err != nil {
```

Comment:
Nit:

```suggestion
		if err := json.Unmarshal(rawBoard, &board); err != nil {
```

Even if the outcome is the same, we don't really want to modify the outer `err` variable.